src/Action/Implementation.hs

```diff
@@ -64,6 +71,14 @@ instance HasSendMail ActionExcept ActionEnv Action where
 instance ActionLog Action where
     log msg = actionIO =<< views envLogger ($ msg)
 
+    readEventLog = do
+        cfg <- viewConfig
+        rows :: [EventLogItemCold]
```

Comment:
Something deletes it from the disk, the location is not mounted, disk is full etc...